cmd/adsysd/integration_tests/adsysctl_policy_test.go

```diff
@@ -583,6 +584,26 @@ func TestPolicyUpdate(t *testing.T) {
 			systemAnswer: "no_proxy_object",
 		},
 
+		// Purge cases
+		"Purge current user policies": {
+			purge:     true,
+			initState: "localhost-uptodate",
+		},
+		"Purge other user policies": {
+			purge:     true,
+			args:      []string{"userintegrationtest@example.com"},
+			initState: "localhost-uptodate",
+		},
+		"Purge machine policies": {
+			purge:     true,
+			args:      []string{"-m"},
+			initState: "localhost-uptodate",
+		},
+		"Purge policies for all connected": {args: []string{"--all"},
```

Comment:
is it still for all connected? it’s now for all users (connected and not connected). I’m unsure if the test setup cover a mix of active/only on cache? Worth testing, probably?

Reply:
oh excellent then! Yeah, renaming will be enough.